framework/behaviors/OptimisticLockBehavior.php

```diff
@@ -32,7 +32,7 @@
  * public function behaviors()
  * {
  *     return [
- *         OptimisticLockBehavior::className(),
+ *         OptimisticLockBehavior::class,
```

Comment:
Fixed in https://github.com/yiisoft/yii2/commit/be8633702d50eee2fecb9158aa877afbd63f8b7b, update branch from 2.1